README.md

```diff
@@ -6,12 +6,29 @@
 The crunchdev package offers a number of convenience functions and [RStudio addins](https://rstudio.github.io/rstudioaddins/) to make developing crunch packages quicker and faster.
 
 ## Installing
-
 The pre-release version of the package can be pulled from GitHub using the [devtools](https://github.com/hadley/devtools) package:
 
     # install.packages("devtools")
     devtools::install_github("Crunch-io/crunchdev")
 
+## Configuration
```

Comment:
I would include three things in the README:
1) a copy-paste example .RProfile 
2) A brief description of how and why you would use  setupCrunch to switch backends
3) A  couple sentences about why this is a useful package.